worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala

```diff
@@ -243,6 +246,16 @@ class FetchHandler(val conf: CelebornConf, val transportConf: TransportConf)
       client.getChannel.writeAndFlush(new ChunkFetchSuccess(req.streamChunkSlice, buf))
         .addListener(new GenericFutureListener[Future[_ >: Void]] {
           override def operationComplete(future: Future[_ >: Void]): Unit = {
+            if (log.isDebugEnabled) {
+              if (future.isSuccess()) {
+                logInfo(
+                  s"Sending ChunkFetchSuccess operation succeeded, chunk ${req.streamChunkSlice}")
+              } else {
+                logError(
```

Comment:
I think we should always print the error log, even it debug is disabled.

Reply:
updated.